### PR TITLE
Arthur kozik hw8

### PR DESCRIPTION
**Pull request by ArthurLimenberg**
Homework

### CODE REVIEW

**Review by katemalash — APPROVED** [review]
Please change the destination branch 
image

**Review by katemalash — CHANGES_REQUESTED** [review]
Please change the destination branch

**Comment by ArthurLimenberg** [comment]
Changed

File: Kozik/HW8/Task1.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+def arithmetic_operation(first_num, second_num, ar_operation):
+    """This function take two numbers and an arithmetic operation (/, *, +, -) and output result"""
+
+    result = 'Unknown operation'
```
Comment (katemalash) [inline Kozik/HW8/Task1.py:4]: It would be nice to have an empty line before 'if' block

File: Kozik/HW8/Task3.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+from Task5 import is_year_leap
```
Comment (katemalash) [inline Kozik/HW8/Task3.py:1]: Nice solution

File: Kozik/HW8/Task4.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+def is_prime(num):
+    """This function determines the primality of a number."""
+
+    for divider in range(2, num):
```
Comment (katemalash) [inline Kozik/HW8/Task4.py:4]: Is it possible to optimize this range?

**Review by katemalash — APPROVED** [review]
Good Job. Please merge this pull request.

**Pull request merged**